example/operator/20-garden.yaml

```diff
@@ -8,15 +8,31 @@ type: Opaque
 stringData:
   hostPath: "/etc/gardener/local-backupbuckets"
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: garden-dns-local
+  namespace: garden
+type: Opaque
+---
 apiVersion: operator.gardener.cloud/v1alpha1
 kind: Garden
 metadata:
   name: local
 spec:
+  dns:
+    providers:
+    - name: local
+      type: local
+    # providerConfig:
+    #   <some-provider-specific-config-for-the-dns-records>
+      secretRef:
+        name: garden-dns-local
   runtimeCluster:
     ingress:
       domains:
-      - ingress.runtime-garden.local.gardener.cloud
+      - name: ingress.runtime-garden.local.gardener.cloud
+      # provider: other-provider # needs to be specified in the `spec.dns.providers` section
```

Comment:
Does this mean that if I don't specify this field, the first provider is chosen? I recommend to augment the comment, it's not clear what happens